test/visualTest.ts

```diff
@@ -123,56 +125,30 @@ describe("DualKpi", () => {
         });
     });
 
-    describe("Capabilities tests", () => {
-        it("all items having displayName should have displayNameKey property", () => {
-            jasmine.getJSONFixtures().fixturesPath = "base";
-
-            let jsonData = getJSONFixture("capabilities.json");
-
-            let objectsChecker: Function = (obj) => {
-                for (let property in obj) {
-                    let value: any = obj[property];
-
-                    if (value.displayName) {
-                        expect(value.displayNameKey).toBeDefined();
-                    }
-
-                    if (typeof value === "object") {
-                        objectsChecker(value);
-                    }
-                }
-            };
-
-            objectsChecker(jsonData);
-        });
-    });
-
     describe("DOM tests", () => {
-        it("svg element created", () => expect(visualBuilder.mainElement[0]).toBeInDOM());
+        it("svg element created", () => expect(visualBuilder.mainElement).toBeDefined());
 
         it("update", (done) => {
             visualBuilder.updateRenderTimeout(dataView, () => {
-                expect(visualBuilder.chartGroup.children("path.area"))
-                    .toBeInDOM();
-                expect(visualBuilder.chartGroup.children("g.axis").children("g.tick").length)
-                    .toBe(4);
+                expect(visualBuilder.pathAreas.length).toBeGreaterThan(0);
+                expect(visualBuilder.axisTicks.length).toBe(4);
                 done();
             });
         });
 
         it("update with null values", (done) => {
-            dataView.categorical.categories[0].values[0] = null;
-            dataView.categorical.values[0].values[0] = null;
+            dataView.categorical!.categories![0].values[0] = null as unknown as PrimitiveValue;
+            dataView.categorical!.values![0].values[0] = null as unknown as PrimitiveValue;
 
             visualBuilder.updateRenderTimeout(dataView, () => {
-                expect(visualBuilder.chartGroup.children("path.area")).toBeInDOM();
+                expect(visualBuilder.pathAreas.length).toBeGreaterThan(0);
                 done();
             }, DefaultWaitForRender);
         });
 
         it("changed data with high contrast on", () => {
-            let dataColorTop: string = getRandomHexColor(),
-                textColorTop: string = getRandomHexColor(),
+            const dataColorTop: string = "#01B8AA",
```

Comment:
add comments, why this hash value is used

Reply:
No particular reason, as it's automated test. The intented change was to remove random color and make the test more stable.